src/css/_colors.scss

```diff
@@ -35,6 +36,7 @@ $colors-light: (
   "secondary-background-dim": #C0C0C0,
   "secondary-foreground": #222222,
 
+  "gray-light": #D3D3D3,
   "gray-mid": #808080,
   "gray-dark": #555555
```

Comment:
Maybe a mixin for the greys that are the same in both?

Reply:
Updated in https://github.com/hubmapconsortium/vitessce/pull/510/commits/b9c2dacd0acbdcf5766d3f89ffda9b01e234aa4c